examples/node/README.md

```diff
@@ -1,9 +1,11 @@
-# React | Typescript | TailwindCSS
+# adCAPTCHA NPM Node package example 
 
 ## Available Scripts
 
 In the project directory, you can run:
 
-### `node api.js`
-
+```bash
+npm start
+```
 
+#### NOTE: `API_KEY` has to be set in your `.env.local` and The node NPM package has to be built running `npm run build`.
```

Comment:
This is fine I guess - but thinking it's weird that you have to build the npm yourself to run an example... perhaps it would have been better to have examples in another repo 🤷  (not suggesting you do this
)

Reply:
Talked about it and its fine as we are using it for dev purposes